db/post_migrate/20210624023831_remove_highest_seen_post_number_from_topic_users.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+require 'migration/column_dropper'
+
+class RemoveHighestSeenPostNumberFromTopicUsers < ActiveRecord::Migration[6.1]
```

Comment:
I am just wondering if it might be best to do this in a separate PR to make sure everything is all good? Otherwise we cannot get the data back if we need to for testing or whatever. Not a huge problem to leave it, just wondering.

Reply:
I have the same question/concern as Martin here, do we need to drop the column now? I see you've added the column to `ignored_columns` with a comment to drop the column in Jan next year, so I'm a little bit confused.

Reply:
Nah, we can easily get the data back if we need it, we can rebuild it from post_timings.